### DISCUSSION TITLE
Is it possible to make Preview work ?

### DISCUSSION THREAD

**Discussion by X901** [skiptools/skip#70] [Q&A] [answered]
Hi, 

I notice that Preview won't work with Skip app
is there a way to make it work ?
or will it work in feature ?

**Comment by marcprux** [comment]
Previews are indeed broken when Skip is enabled. I've logged this as https://github.com/skiptools/skip/issues/72. 

Until we get a proper fix in place, you can use the workaround described in that issue report, which involves editing the `Run skip gradle` phase of the Build Phases for the project.

Thanks reporting it.

**Comment by marcprux** [comment, marked as answer]
Update: Skip now works with Xcode previews! https://github.com/skiptools/skip/issues/72 was fixed, so any new projects you create with `skip init` should work with previews.

![image](https://github.com/skiptools/skip/assets/659086/4f7f24d7-c195-40dd-a587-06db0584ef2f)
